package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
We should keep this consistent with https://github.com/infracost/actions/blob/master/package.json. Especially the LICENSE.